website/redirects.js

```diff
@@ -108,6 +108,11 @@ module.exports = [
     destination: '/boundary/docs/install-boundary/no-gen-resources',
     permanent: true,
   },
+  {
+    source: '/boundary/docs/install-boundary/no-gen-resources',
+    destination: '/boundary/docs/install-boundary/initialize',
+    permanent: true,
+  },
```

Comment:
Wasn't sure if this was needed

Reply:
Yes! Good catch. I just pushed a commit to add the other files that were moved under the new **Architecture** folder. They need redirects too.

I will review the rest of this content next.